### PR TITLE
Minor changes in section: Books (english) - Python - Pandas

### LINKED ISSUE #5431: Pandas entry update on Books - English - Python?
Hello contributors 👋

Is really needed have this entry with "in process" note? It always points to last version 😉.
https://github.com/EbookFoundation/free-programming-books/blob/62da834ed2d7601d50fe6e7f51b5db41568d1e0e/books/free-programming-books.md#L2570-L2581

I can contribute with this minor change. Or updating to add html links:

* Online: https://pandas.pydata.org/docs
* HTML: https://pandas.pydata.org/docs/pandas.zip
* PDF: https://pandas.pydata.org/docs/pandas.pdf (already added) 👆

How to proceed in this case?

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Improve repo

### Description
Minor changes in section: Books (english) - Python - Pandas

### Why is this valuable (or not)?
Resolves #5431, Closes #5431

### How do we know it's really free?
N/A. It's the official site.

### For book lists, is it a book? For course lists, is it a course? etc.
Books

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books.md [hunk 1]
```diff
@@ -2575,7 +2575,9 @@ Kerridge (PDF) (email address *requested*, not required)
 #### Pandas
 
 * [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.18.1)
-* [pandas: powerful Python data analysis toolkit](https://pandas.pydata.org/docs/pandas.pdf) - Wes McKinney and the Pandas Development Team (:construction: *in process*) (PDF)
+* [pandas: powerful Python data analysis toolkit](https://pandas.pydata.org/docs) - Wes McKinney and the Pandas Development Team
+  * [HTML zipped format](https://pandas.pydata.org/docs/pandas.zip) (HTML)
```
Comment (davorpa) [inline books/free-programming-books.md:2579]: Is enough this or I should remove HTML/PDF?

Comment (eshellman) [reply books/free-programming-books.md:2579]: usually best to have one link go to a page where multiple formats are available. Sometimes that doesn't exist, or it has multiple books.

Comment (davorpa) [reply books/free-programming-books.md]: Done!

**Comment by eshellman** [comment]
super

**Pull request merged**